commit c6cbb1e830e21be25b21a21966f7e46a577b1a51 (refs/changes/00/5400/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2019-01-22T09:40:57-08:00 (9 months ago)
    
    TRITON-788 sdc-amonadm breaks with sdc-clients@11.x requirement of a version option to SAPI client
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index deae0d7..1998bc0 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,14 +1,19 @@
 # sdc (SDC ops core zone) Changelog
 
+## 1.7.1
+
+- TRITON-788 Update to sdc-amonadm that isn't broken by sdc-clients@11
+  change.
+
 ## 1.7.0
 
 - TRITON-1076 remove unused data dumps from dump-hourly-sdc-data.sh
 - Dumps that remain (all others were removed):
-    * cnapi_servers-$TIMESTAMP.json
-    * napi_networks-$TIMESTAMP.json
-    * papi_packages-$TIMESTAMP.json
-    * vmadm_vms-$TIMESTAMP.json
-    * vmapi_vms-$TIMESTAMP.json
+    * `cnapi_servers-$TIMESTAMP.json`
+    * `napi_networks-$TIMESTAMP.json`
+    * `papi_packages-$TIMESTAMP.json`
+    * `vmadm_vms-$TIMESTAMP.json`
+    * `vmapi_vms-$TIMESTAMP.json`
 
 ## 1.6.1
 
diff --git a/package.json b/package.json
index c07153b..0e4ecac 100644
--- a/package.json
+++ b/package.json
@@ -1,12 +1,12 @@
 {
     "name": "sdc",
     "description": "SmartDataCenter 'sdc' zone to hold ops/admin tools",
-    "version": "1.7.0",
+    "version": "1.7.1",
     "author": "Joyent (joyent.com)",
     "private": true,
     "dependencies": {
         "sdc-clients": "8.1.2",
-        "amonadm": "git+https://github.com/joyent/sdc-amonadm.git#fcf2fd8",
+        "amonadm": "git+https://github.com/joyent/sdc-amonadm.git#v1.0.1",
         "smartdc": "git+https://github.com/joyent/node-smartdc.git#4308fd9",
         "manta": "git+https://github.com/joyent/node-manta.git#b277acf",
         "moray": "git+https://github.com/joyent/node-moray.git#b84ef0e",
